### PR TITLE
created art pieces list and list item

### PR DESCRIPTION
**Pull request by DanielBerneisch**
please review

### CODE REVIEW

**Comment by UmaySoyluer** [comment]
looks nice

File: components/ArtPieces/ArtPieces.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import ArtPiecePreview from "../ArtPiecePreview/ArtPiecePreview";
+import Image from "next/image";
+
+export default function ArtPieces({ artPieces }) {
+  console.log(artPieces[0].name);
```
Comment (rythinks) [inline components/ArtPieces/ArtPieces.js]: left-over console.log - should probably be removed :)

File: components/ArtPiecePreview/ArtPiecePreview.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Image from "next/image";
+
+export default function ArtPiecePreview({ img, name, artist }) {
```
Comment (rythinks) [inline components/ArtPiecePreview/ArtPiecePreview.js]: fyi: the names of the props to not match the suggested / requested naming in the user story:
`ArtPieces` props: `pieces`
`ArtPiecePreview` props: `image`, `title`, `artist`

**Review by rythinks — COMMENTED** [review]
only had two suggestions - everything else looks good to me :)

File: components/ArtPiecePreview/ArtPiecePreview.js (hunk 2 above)
Comment (DanielBerneisch) [reply components/ArtPiecePreview/ArtPiecePreview.js]: changed they prop names to the names given in the user story

**Pull request merged**